### ISSUE TITLE
Deleting scan location on mobile, doesn't actually delete scan location

### ISSUE THREAD

**Issue by dwenzier** [favll/pogom#226]
## Checklist
 - [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
 - [x] I have checked if this issue already exists (both open and closed issues)
 - [x] I am using the most recent version of pogom from the master branch
 - [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
 - [ ] I'm posting the complete log
 - [x] I'm using a descriptive title for this issue

## My Issue

So I've setup the server with a public IP and open port. 
I was able to see the scanned map on my mobile, worked perfect.
Now as I traveled I removed the old scan radius and added a new one, where I was at that moment.
After some time it wouldn't show any Pokemon anymore. 

Back home I saw why: every scanner I deleted on mobile, didn't delete on the map at home.

Scanner at home working on Windows 10 in Chrome.
Browser on mobile, Chrome on Android.

tl;dr: Can add scanners and remove them on Mobile. Will add them on server at home, but won't remove them.

**Comment by nborrmann** [comment]
Were you able to add locations?

**Comment by dwenzier** [comment]
Yes, I was able to add and remove them on Mobile.
When I looked at the map at home, they were all added, but not removed.

**Comment by KikkomanChow** [comment]
I've encountered the same issue. I'm on Safari on mobile. (iphone)

**Comment by CaptorOfSin** [comment]
Isn't this just a visual bug which is resolved by reloading the browser?
I've seen several times that all clients pick up added locations, but only the issuing client will also get feedback on removed locations.
The locations are removed correctly on the server though, just markers still visible on clients.

**Comment by TheHippo** [comment]
> Isn't this just a visual bug which is resolved by reloading the browser?

No. At least for me. After deleting a location it's gone until I reload the page. Then its back on again.

Sometimes there is a similar issue with adding new locations. (They appear only after refreshing.)
